data/human/free worlds 0 prologue.txt

```diff
@@ -1173,7 +1173,7 @@ mission "FW Katya 1"
 	destination "Glory"
 	to offer
 		or
-			random < 10 + 7 * "assisted free worlds" * "assisted free worlds"
+			random < 70 + 7 * "assisted free worlds" * "assisted free worlds"
```

Comment:
```suggestion
			random < 10 + 15 * "assisted free worlds" * "assisted free worlds"
```

This would be far more appropriate, but may even still be too drastic.

Reply:
I personally dont mind it the way it is, no reason to rush it. It already offers in all of the Free World.
It being this way gives you more time to do other things.

Reply:
The writing also is assuming it's either the first or second FW mission chain you will complete, since when you're offered a commission it goes "Katya must still be undercover." Which is a dumb thing to say when it's the third one and she's been undercover for like three days. 

Nobody makes you complete the mission once you get it. But it being one of the last ones you get doesn't make much sense with how it's written. 

It doesn't have to be a septuple chance. But it should, IMO, be offering before the others, or at LEAST as soon as them. Which would be a base 35% chance, minimum. 10% is far too low for something that fundamental and which is written as being done early. 

Having the Pact missions make you count as more helpful to the Free Worlds so she approached you earlier if you did those might also help, though that's getting more involved.

Reply:
That makes it a little too common for it to trigger. EG:

`random < 10 + 7 * "assisted free worlds" * "assisted free worlds"` is <10 plus 7 times the number of assisted free worlds you've done squared. So if you complete 4 afw's you should be at  `random < 112`.

`random < 70 + 7 * "assisted free worlds" * "assisted free worlds"` would be 310.

But https://github.com/endless-sky/endless-sky/wiki/Player-Conditions#reserved-conditions says that `random` is a number between 1 and 99, so the 70+7*afw^2 means that you have ~ a 70% chance of getting Katya without ever doing an AFW, 77% after 1, 98% after 2, and 100% after 3. The other alt (10+15 etc) would mean 10% for zero, 25% for 1, 70% for 2, 100% for 3.

It's still a bit too generous. 

`random < 10 + 9 * "assisted free worlds" * "assisted free worlds"` would give you AFW 0=10%, 1=19%, 2=46% 3=91%, 4=100%

Reply:
10% base is a ridiculously low chance for this mission. Katya's not dragging her heels about investigating nuclear explosions. And it doesn't need a ton of combat, which the other ones all do - gatekeeping it behind those makes the writing make less sense. It makes sense early, not late. 

The other missions take a significant amount of time to complete. For example, completing the convoy chain and getting that +1 takes a minimum of nine weeks (for most of which you won't be in the FW, so can't get Katya's mission). If it is spawning after other missions, it's going to mean the explosion investigation takes years, which makes the plot silly. 

There's also the bit that Katya approaches you as if you're someone she has no familiarity with at all. The more missions you've done for the FW, the less sense that makes. And where you are given repeated options to be uncomfortable and unwilling to work for the FW, which is pointless if we're only going to give this to people who do that a lot. 

It also works much better narratively if she's dropped off, spends time undercover, you do other things, wondering what the investigation will produce, are offered a commission, join...and then the Republic invades. 

It absolutely should not be last. 

Having just redone the path, there's nothing inconvenient about it. You occasionally have two mandatory passengers or some cargo. No combat, bar the surveillance drone. And it pays extremely well. As an intro to the cause of the FW, it works very well, and it also does a lot to convince the players they're not terrorists, which seems important from a roleplaying perspective.

Reply:
I agree that 10% base rate is way too low. However, the narrative reason seems to be that you've interacted with the faction at least once before beginning the faction plot. 

Three ways of solving it would be
`random < 10 + 9 * "assisted free worlds" * "assisted free worlds"` gives you AFW 0=10%, 1=19%, 2=46% 3=91%

`random < 30 + 7 * "assisted free worlds" * "assisted free worlds"` gives you AFW 0=30%, 1=37%, 2=58% 3=93%

Or my personal favourite is 
`random < 30 + 8 * "assisted free worlds" * "assisted free worlds"` gives you AFW 0=30%, 1=38%, 2=62% 3=100%

This one gets you a strong chance of getting Katya mission at first arrival or after one AFW, almost a 2/3 chance after two, and a guaranteed chances on completion of the third quest chain.

Using 40+7*AFW^2 would work really well also.

Anything more than a 50% chance of getting the Katya mission without having done a single AFW seems counter to the future idea of having three main plots.

(I just realized that the fourth AFW is for completing Katya, so I'm only using 0,1,2,3 in my estimates now.)

Reply:
That's a really good point about the pact missions. I wonder if having "Pact Recon 3" end with an AFW would break anything else.

Reply:
I think the major problem with it might be that they would end up offering you a commission sooner (increasing the chances that you miss some of the pre-commission helping them out content). Other than that, I would think it should be fine.